docs/source/connecting/connecting.md

```diff
@@ -35,6 +35,12 @@ async fn main() -> Result<(), Box<dyn Error>> {
 After successfully connecting to some specified node the driver will fetch topology information about
 other nodes in this cluster and connect to them as well.
 
+Driver maintains this pool of connections and each connection is capable of handling multiple requests in parallel. Driver will also route the request to node / shard that actually owns the data (unless load balancing policy that you use doesn't support it).
```

Comment:
`Driver maintains this pool`? Maybe `a pool`, or `its own pool`?

Reply:
Also, `The driver` instead of `Driver` sounds slightly better

Reply:
`load balancing policy` -> `the load balancing policy`

Reply:
`the request` -> `requests`?

Reply:
> `the request` -> `requests`?

Changed to `Driver will also route requests to nodes / shards that actually own the data (unless the load balancing policy that you use doesn't support it).`

Reply:
> `Driver maintains this pool`? Maybe `a pool`, or `its own pool`?

I was referring to a pool that was mentioned in a sentence before. But now that I think of it your version may be easier to understand.
Changed to `The driver maintains its own pool of connections to each node and each connection is capable of handling multiple requests in parallel.`